src/spec/page-objects/billing-page.js

```diff
@@ -3,6 +3,8 @@ class BillingPage {
     this._url = '/#/settings/billing';
     this._planName = $('.billing--plan-name-container .plan-name');
     this._planPrice = $('.billing--plan-name-container .plan-price');
+    this._ccNumber = element(by.id('cc.number'));
+    this._ccIcon = $('.icon');
```

Comment:
When we add another icon into that view i think this will generate a problem, can you try to set a class into the icon you're trying to get so it's less error-prone.